_source/logzio_collections/_log-sources/mongo.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Ship MongoDB logs
+logo:
+  logofile: mongo-logo.png
+  orientation: vertical
+data-source: MongoDB
+data-for-product-source: Logs
+templates: [beats-logs]
+contributors:
+  - nshishkin
+shipping-tags:
+  - database
+order: 120
+---
+MongoDB is a source-available cross-platform document-oriented database program. Fluentd is an open source data collector and a great option because of its flexibility. This integration lets you send logs from your MongoDB instances to your Logz.io account using Fluentd.
+
+
+#### Step by step
+
+**Before you begin, you'll need**:
+
+* MongoDB installed on your host
+
+<div class="tasklist">
+
+
+##### Configure MongoDB to write logs to a file
+
+In the configuration file of your MongoDB instance, set the database to write logs to a file. You can skip this step if this has already been configured on your MongoDB host.
+
+To do this, add the following to your MongoDB configuration file:
+
+
+```yaml
+systemLog:
+   destination: file
+   path: "<<MONGODB-FILE-PATH>>"
+   logAppend: true
+```
+
+* Replace `<<MONGODB-FILE-PATH>>` with the path to the log file, for example, `/var/log/mongodb/mongod.log`.
+
```

Comment:
If we're handling the log file here, maybe it's makes more sense to add the step of the "chmod" command after that?